src/components/common/ListShareClass.vue

```diff
@@ -39,7 +39,23 @@
         disable-pagination
         disable-sort
         hide-default-footer
+        hide-default-header
       >
+        <template
+          #header="{ props: { headers } }"
+        >
+          <thead>
+            <tr>
+              <th
+                v-for="(h, index) in headers"
+                :key="index"
+                :class="h.class"
+              >
+                <span>{{ h.text }}</span>
+              </th>
+            </tr>
+          </thead>
+        </template>
```

Comment:
As this is a common component, this is how it looks when just a list:
Before/DEV:
![image](https://github.com/user-attachments/assets/48cce1ce-8810-4443-8ea5-f31b664a2048)

After:
![image](https://github.com/user-attachments/assets/dbefe9e4-c646-455f-8e08-07e238203f21)

When in edit mode:
Before/DEV:
![image](https://github.com/user-attachments/assets/aa9ca74f-4656-4409-a9ca-d6f385937ff4)

After:
![image](https://github.com/user-attachments/assets/0a9057bf-df44-4e5f-a4b1-34926cb41339)

Reply:
Awesome! Looks great.